GoogleForCreators/web-stories-wp
# Video Optimization: Add missing `crossorigin` attributes

## Context



## Summary

This is a follow-up to #6875 .

With these new headers, 3P media are no longer loaded due to missing crossorigin attributes.

## Relevant Technical Choices

I have also improved poster images, by making sure, loading=lazy, alt and onload are loaded on poster images as well. 

This PR will likely fix another issue, where images from media library are served from another url, like if assets are loaded from s3.

## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Enable video optimization experiment
2. Browsing and using 3P Media (videos, images, GIFs) should work without issues or console errors  
    **Note:** there can be some console errors for GIFs due to caching, but most of them should load fine already.
3. Toggling "show images" in Page templates tab should not cause console errors

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7016
